frontend/test/src/LiveMessageManager.spec.js

```diff
@@ -4,25 +4,45 @@ import {
   LIVE_MESSAGE_STATUS
 } from '../../src/util/LiveMessageManager.js'
 
+import { enforceOptions } from 'broadcast-channel'
+
+// RJ - 2020-09-13 - NOTE: Makes the tests run faster.
+// See https://github.com/pubkey/broadcast-channel#enforce-a-options-globally
+enforceOptions({ type: 'simulate' })
+
 const apiUrl = 'http://localhost/api'
 const userId = 1
 
+// RJ - 2020-09-13 - NOTE: The next two methods are used to track status changes
+// in this test suit.
+
+const setStatusOrig = LiveMessageManager.prototype.setStatus
+LiveMessageManager.prototype.setStatus = function (status) {
+  setStatusOrig.call(this, status)
+
+  const resolve = (this.statuscallbacks && this.statuscallbacks.shift())
+  resolve && resolve()
+}
+
+LiveMessageManager.prototype.onstatuschange = function (callback) {
```

Comment:
shouldn't you name this function in camel case ?

Reply:
I figured I would follow the case for the event handlers used everywhere on the DOM. Are you fine with this or should I change this?

(onmousemove, onreadystatechange, etc)

Reply:
mmh it makes sense but it also surprised me to see all your new functions in camel case but this one.
I'd say to put it in camel case to match all the others functions name of the file

Reply:
ok :-)